easybuild/easyconfigs/k/Kent_tools/Kent_tools-479-GCC-13.3.0.eb

```diff
@@ -0,0 +1,43 @@
+easyblock = 'MakeCp'
+
+name = 'Kent_tools'
+version = '479'
+
+homepage = 'https://genome.cse.ucsc.edu/'
+description = """Kent utilities: collection of tools used by the UCSC genome browser."""
+
+toolchain = {'name': 'GCC', 'version': '13.3.0'}
+source_urls = [
+    'https://hgdownload.cse.ucsc.edu/admin/exe/',
+    'https://hgdownload.cse.ucsc.edu/admin/exe/userApps.archive/',
+]
+sources = ['userApps.v%(version)s.src.tgz']
+
+dependencies = [
+    ('MariaDB', '11.7.0'),
+    ('libpng', '1.6.43'),
+    ('zlib', '1.3.1'),
+    ('util-linux', '2.40'),
+    ('OpenSSL', '3', '', SYSTEM),
+    ('freetype', '2.13.2'),
+]
+
+
+prebuildopts = "sed -i 's/#include <algorithm>/#include <algorithm>\\n#include <cstdint>/g' "
```

Comment:
can you please add a comment here explaining this inclusion? this looks like it is related to a chnage in GCC 13, so maybe it will be fixed in future versions of `Kent_tools` and then it can be removed.